src/mmw/js/src/modeling/gwlfe/runoff/views.js

```diff
@@ -75,6 +77,10 @@ var ResultView = Marionette.LayoutView.extend({
         }
     },
 
+    onRender() {
```

Comment:
Good catch -- I meant to replace these before pushing so I'll make a fixup commit for that.